GooglePlayInstant/Editor/PlayInstantEditorMenu.cs

```diff
@@ -68,21 +69,31 @@ private static void SetUpPlayInstantSdk()
             PlayInstantSdkInstaller.SetUp();
         }
 
-#if UNITY_2018_3_OR_NEWER
-        [MenuItem("PlayInstant/Build Android App Bundle...", false, 300)]
-        private static void BuildAndroidAppBundle()
+        [MenuItem("PlayInstant/Quick Deploy/Overview...", false, 300)]
```

Comment:
Should Quick Deploy be moved into the previous section, e.g. as "203"? It kind of seem strange to me to have the horizontal dividers above and below a single menu item.

Reply:
That's fair, I'll put it in the same section as "Build Settings..." and "Set Up Instant Apps Development SDK..."